data/api/client-server/third_party_lookup.yaml

```diff
@@ -98,12 +98,14 @@ paths:
           schema:
             type: string
         - in: query
-          name: searchFields
+          name: fields
```

Comment:
Why change the name? `searchFields` looks appropriate.

Reply:
I think it makes sense to use the same name everywhere:
- https://spec.matrix.org/v1.11/client-server-api/#get_matrixclientv3thirdpartyuserprotocol
- https://spec.matrix.org/v1.11/application-service-api/#get_matrixappv1thirdpartylocationprotocol
- https://spec.matrix.org/v1.11/application-service-api/#get_matrixappv1thirdpartyuserprotocol

For some reason the exact same API on the appservice endpoints uses `fields` and not `searchFields`. And the similar API to find users for a protocol also uses `fields` on both the CS and AS API. I think it makes sense to be consistent there, but I can revert that part, if you would prefer that.